tinker/templates/e-announcements/results.html

```diff
@@ -47,10 +47,35 @@
     <div class="content">
         <h2>{{ forms_header }}</h2>
         {% if search_results is defined and search_results | length > 0 %}
-            {% for form in search_results %}
-                <br/>
-                {{ form_card(form) }}
-            {% endfor %}
+            <div id="loop" class="form-group col-md-12">
+                <nav aria-label="Page navigation example">
+                    <ul class="pagination">
+                        <li id="previous-page">
+                            <a href="javascript:void(0)">
+                                <span aria-hidden="true">&laquo;</span>
+                            </a>
+                        </li>
+                    </ul>
+                    <ul>
+                        <li style="list-style-type: none;">
+                            Limit
+                        </li>
+                        <li style="list-style-type: none;">
+                            <select id="selected-option">
+                                <option id="first" selected="selected" value="10">10</option>
```

Comment:
the "first" id here is bad :)

Reply:
Changed to ```selected-default``` since it is the option that is set to selected to start.